test/fixtures/requests/prepare-suspended-payment-execution.json

```diff
@@ -3,5 +3,5 @@
   "suspensionSequence": 1234,
   "method": 1,
   "digest": "8F434346648F6B96DF89DDA901C5176B10A6D83961DD3C1AC88B59B2DC327AA4",
-  "proof": "whatever"
+  "proof": "whateverthis bemusthave 32 chars"
```

Comment:
Does the digest match this proof? Could you make it a little cleaner like "this must have 32 characters...."?